### PR TITLE
OSDOCS-12169: adds 4.14.43 relnotes Microshift

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.14

Issue:
[OSDOCS-12169](https://issues.redhat.com/browse/OSDOCS-12169)

Link to docs preview:
[4-14-43-dp_release-notes](https://86302--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-14-release-notes.html#microshift-4-14-43-dp_release-notes)

QE review:
- NA. stock text updated. No other changes.

### CODE REVIEW

**Comment by aravipra** [comment]
Only Peer review is need for now. Please do not merge unless the label is added.

**Comment by snarayan-redhat** [comment]
> lgtm
> 
> @aravipra
> 
> The advisory links seem to be not working. I got a 404 error when I opened the following links:
> 
> * https://access.redhat.com/errata/RHBA-2024:11033
> * https://access.redhat.com/errata/RHSA-2024:11031
> 
> image
> 
> Requesting @snarayan-redhat for another round of peer review, as I've recently joined the peer review squad.

@aravipra I believe the links will work post GA?

**Comment by aravipra** [comment]
> > lgtm
> > @aravipra
> > The advisory links seem to be not working. I got a 404 error when I opened the following links:
> > 
> > * https://access.redhat.com/errata/RHBA-2024:11033
> > * https://access.redhat.com/errata/RHSA-2024:11031
> > 
> > image
> > Requesting @snarayan-redhat for another round of peer review, as I've recently joined the peer review squad.
> 
> @aravipra I believe the links will work post GA?

The links will work once the advisory is published. Only after advisory is published this PR will be merged. Hence the links will work post that. So the links are correct.

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 1]
```diff
@@ -665,4 +665,13 @@ Issued: 5 December 2024
...
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-14-43-dp_{context}"]
+=== RHBA-2024:11033 - {microshift-short} 4.14.43 bug fix and security update advisory
+
+Issued: 18 December 2024
+
+{product-title} release 4.14.43 is now available. The list of enhancements and bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:11033[RHBA-2024:11033] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:11031[RHSA-2024:11031] advisory.
+
```
Comment (sr1kar99) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:676]: /lgtm

**Review by sr1kar99 — COMMENTED** [review]
/lgtm

**Review by xenolinux — COMMENTED** [review]
merge review LGTM

**Pull request merged**